### ISSUE TITLE
Improve success check for quantization filter (i.e., BitGroom) test-suite

### ISSUE THREAD

**Issue by czender** [ccr/ccr#48]
Quantization alters data, so tests do not check for equality. Currently implemented with loose tests like
```
		 if (round(data_in[x][y]) <= data_out[x][y]-2 ||
		     round(data_in[x][y]) >= data_out[x][y]+2 ) ERR;
```
Replace with stringent tests like  `round((x*10^NSD)/10^NSD)`

**Comment by czender** [comment]
Fixing this via #71

**Comment by czender** [comment]
Fixed #48 via #71
